my-webxr-app/src/utils/CsvUtils.tsx

```diff
@@ -79,6 +81,8 @@ export async function parseAndHandleLocalCsv(
       if (batch.length > 0) {
         await handleParsedData(batch, dbName, storeName, i);
       }
+      const rollbar = new Rollbar(rollbarConfig);
+      rollbar.debug('Loaded CSV successfully');
```

Comment:
Is this for the testing team to run automated tests? Otherwise do we need this if we have a message on screen displaying that it was successful?

Reply:
This is an example of using Rollbar logs outside of React. I want to do a minor refactor on this though—I think I've got a better way to accomplish this.